docs/http/traffic-policy/gallery.mdx

```diff
@@ -72,3 +72,54 @@ This rule applies rate limiting of `30` requests per second to the endpoint `/ap
 		],
 	}}
 />
+
+## Enforce Basic Auth
+
+This rule mimics the behavior of the [Basic Auth module](/http/basic-auth/). Requests that are missing the Authorization header or contain an invalid authorization value will receive an unauthorized response, prompting the requester to authorize themselves.
```

Comment:
I think we should also include the caveat that it's less secure.

The basic auth module stores your password hashed in the database, ensuring ngrok doesn't store the plaintext at rest.
This also has the side-effect of making login attempts slower (since we hash the incoming password too for checking), meaning brute-forcing is harder for an attacker.

Using policies necessitates us storing the password plaintext, and as such is suitable for demo and prototyping uses, but not for most prod uses.

It's probably worth having a big warning blurb in this section of "this stores the password in plaintext and should only be done if you understand the security implications of this" so we don't seem like we're irresponsible with respect to security.